network-config-analyzer/HelmScanner.py

```diff
@@ -0,0 +1,81 @@
+#
+# Copyright 2022- IBM Inc. All rights reserved
+# SPDX-License-Identifier: Apache2.0
+#
+
+"""
+Utility class to control HELM
+"""
+
+import os
+from CmdlineRunner import CmdlineRunner
+import re
+
+
+class HelmScanner:
+    """
+       A class for handling HELM and resolving HELM Charts
+    """
+
+    def __init__(self):
+        self.template_files = []
+
+    def parse_chart(self, chart_dir):
+        """
+        Resolve HELM Chart file and parse it for yamls.
+        :param chart_dir: The path of the Chart package
+        :return: dict: dict of yaml filenames and their resolved content
+        """
+        resolved_yamls_dict = {}
+        resolved_yamls = CmdlineRunner.resolve_helm_chart(chart_dir)[:-1]
+        resolved_yamls = str(resolved_yamls).encode('ascii', 'ignore').decode('unicode_escape')
+        resolved_yamls = resolved_yamls[:-1]
+        resolved_yamls_list = str(resolved_yamls).split('---')[1:]
+
+        # insert Chart and Values into the templates list, so they will be not parsed again.
+        self.template_files.append(os.path.join(chart_dir, 'Chart.yaml'))
+        self.template_files.append(os.path.join(chart_dir, 'values.yaml'))
+
+        for index, file in enumerate(resolved_yamls_list):
+            file_name = re.findall('# Source: (.*\\.yaml)', file)[0]
+
+            # create the full path from the chart dir
+            file_name_idx = str(file_name).find('/')
+            file_name = file_name[file_name_idx + 1:]
+            file_name = os.path.join(chart_dir, file_name)
+            file_name = file_name.replace('/', '\\')
```

Comment:
what's that for? Isn't this Windows specific (using '\\' in paths instead of '/')?